### ISSUE TITLE
I can successfully run this code but not the right result

### ISSUE THREAD

**Issue by yongxuUSTC** [basveeling/wavenet#23]
I download the VCTK datasets, there are many folds for each speaker. I just copy the data of one speaker into the "train" & "test" dir (should be mkdir train & mkdir test)

And then i run like this:
python wavenet.py with 'data_dir=/vol/vssp/datasets/audio/dcase2016/vctk_data/VCTK-Corpus/wav48' small adam

(Before that: change this:
#def small(desired_sample_rate):
def small():
    desired_sample_rate = 4000)

And then **(but i can not generate the wanted wav during test. Any wrong with my trianing process ???)**
WARNING - root - Changed type of config entry "optimizer.epsilon" from NoneType to float
INFO - wavenet - Running command 'main'
WARNING - wavenet - No observers have been added to this run
INFO - wavenet - Started
Configuration (modified, added, typechanged):
  batch_size = 16
  data_dir = '/vol/vssp/datasets/audio/dcase2016/vctk_data/VCTK-Corpus/wav48'
  data_dir_structure = 'flat'
  debug = False
  desired_sample_rate = 4000
  dilation_depth = 8
  early_stopping_patience = 20
  final_l2 = 0
  fragment_length = 640
  fragment_stride = 400
  keras_verbose = 1
  learn_all_outputs = True
  nb_epoch = 1000
  nb_filters = 16
  nb_output_bins = 256
  nb_stacks = 1
  predict_initial_input = ''
  predict_seconds = 1
  predict_use_softmax_as_input = False
  random_train_batches = False
  randomize_batch_order = True
  res_l2 = 0
  run_dir = None
  sample_argmax = False
  sample_temperature = 1.0
  seed = 348190421
  test_factor = 0.1
  train_only_in_receptive_field = True
  train_with_soft_target_stdev = None
  use_bias = False
  use_skip_connections = True
  use_ulaw = True
  optimizer:
    decay = 0.0
    epsilon = 1e-08
    lr = 0.001
    momentum = 0.9
    nesterov = True
    optimizer = 'adam'
INFO - main - Running with seed 348190421
INFO - main - Loading data...
INFO - main - Building model...
INFO - build_model - Receptive Field: 512 (128ms)
................................................................(model.summary).............................................
INFO - main - None
INFO - main - Compiling Model...
INFO - main - Starting Training...
Epoch 1/1000
9632/9632 [==============================] - 196s - loss: 4.3700 - categorical_accuracy: 0.0332 - categorical_mean_squared_error: 2048.7681 - val_loss: 4.3286 - val_categorical_accuracy: 0.0515 - val_categorical_mean_squared_error: 3280.6926
Epoch 2/1000
9632/9632 [==============================] - 198s - loss: 3.8852 - categorical_accuracy: 0.0808 - categorical_mean_squared_error: 2505.4959 - val_loss: 4.2130 - val_categorical_accuracy: 0.0652 - val_categorical_mean_squared_error: 3624.2373
Epoch 3/1000

**Comment by basveeling** [comment]
Have you tried using the vctkdata namedconfig? E.g. 
```bash
python wavenet.py with 'data_dir=/vol/vssp/datasets/audio/dcase2016/vctk_data/VCTK-Corpus/wav48' small adam vctkdata````
